tasks/closedSourceRationaleCheck.ts

```diff
@@ -0,0 +1,39 @@
+import { danger } from "danger"
+
+export const target = "Rationale for Closed Source"
+export const message = "missing rationale for closed source"
+
+const parseReadme = async (getter, args) => {
+  const { data: contents } = await getter(args)
+  const readme = Buffer.from(contents.content, "base64").toString("utf8")
+  return readme
+}
+
+const findMissing = (repos, getter, org) => {
+  const missingRationale = repos.filter(async repo => {
+    try {
+      const args = { owner: org, repo: repo.name, path: "README" }
+      const readme = await parseReadme(getter, args)
```

Comment:
I'd recommend using [`danger.github.utils.getContents`](https://danger.systems/js/reference.html#GitHubUtilsDSL) instead. It's basically the code above but a missing file is an empty string with is falsy in JS